README.md

```diff
@@ -76,7 +77,7 @@ A <a id="local-file" href="#local-file">local file</a> path consists of one or m
 
 ---
 
-A content block <a id="content-block-title" href="#content-block-title">title</a> consists of of either:
```

Comment:
Thank you for spotting this! Since it’s not related to fragment identifiers in any way, please remove this from the pull request. If you’d like to submit another pull request, let me know. Otherwise I can commit this fix directly to the branch.